src/test/regress/upgrade/utils.py

```diff
@@ -3,16 +3,9 @@
 from config import USER
 
 
-def run(command):
-    return subprocess.call(command)
-
-def run_with_stdout(command, file):
-    return subprocess.call(command, stdout = file)
-
-
 def psql(pg_path, port, command):
-    return run([os.path.join(pg_path, 'psql'), '-U', USER, '-p', str(port),
-                '-c {}'.format(command)])
+    return subprocess.call([os.path.join(pg_path, 'psql'), '-U', USER, '-p', str(port),
+                            '-c {}'.format(command)])
```

Comment:
wouldn't it be possible to do this without formatting?
```python
[...
'-c', command
]
```

Reply:
it is definitely possible :)